cmd/authelia-scripts/docker.go

```diff
@@ -50,3 +50,8 @@ func (d *Docker) Manifest(tag, amd64tag, arm32v7tag, arm64v8tag string) error {
 func (d *Docker) CleanTag(username, password, tag string) error {
 	return CommandWithStdout("curl", "-u", username+":"+password, "-X", "DELETE", "https://cloud.docker.com/v2/repositories/"+DockerImageName+"/tags/"+tag+"/").Run()
 }
+
+// PublishReadme push README.md to dockerhub.
+func (d *Docker) PublishReadme(username, password string) error {
+	return CommandWithStdout("docker", "run", "--rm", "-e DOCKERHUB_USERNAME="+username, "-e DOCKERHUB_PASSWORD="+password, "-e GIT_REPOSITORY="+DockerImageName, "-e DOCKER_REPOSITORY="+DockerImageName, "lsiodev/readme-sync", "bash", "-c", "node sync").Run()
```

Comment:
A straight API call would be better for security reason. I don't want to give my credentials to a docker container we have not audited and built ourselves (I mean built by Authelia CI from source Dockerfile).